models/Stable-diffusion/readme.md

```diff
@@ -0,0 +1,7 @@
+ * You have to install `checkpoints` models in this folder for `Stable Diffusion`.
+ * You can find the models in [file storage][ref-fstor] or [the original link][ref-original].
+ * The original model version requires you to share informatoion with [`huggingface.co`][ref-hugface]!
+
+[ref-original]: https://huggingface.co/CompVis/stable-diffusion-v-1-4-original
+[ref-fstor]: https://drive.yerf.org/wl/?id=EBfTrmcCCUAGaQBXVIj5lJmEhjoP1tgl
+[ref-hugface]: https://huggingface.co
```

Comment:
Totally understand the value of separating ref links like this for maintenance purposes, but people will be opening these files in text editors on their local machines, most likely not rendering the markdown. Maybe use these URLs directly to the links that are referencing them so users don't need to do the referencing in their heads (not everyone understands how markdown or link referencing works).

Reply:
Nope. Most text editors have addons for MD anyway..